librz/il/definitions/bool.h

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef BUILD_BOOL_H
+#define BUILD_BOOL_H
+#include "common.h"
+
+struct bool_t {
+	bool b;
+};
+typedef struct bool_t *Bool;
+
+RZ_API Bool rz_il_new_bool(bool true_or_false);
+RZ_API Bool rz_il_bool_and_(Bool a, Bool b);
```

Comment:
Why the `_` suffix?

Reply:
It existed in the original CoreTheory Signature

Reply:
In the original CT it was `and_` because the `and` is the OCaml keyword. Thus, we can remove the `_` from the name.